packages/icons/src/StatusIcon.tsx

```diff
@@ -10,24 +10,29 @@ type StatusIconProps = IconProps & {
 };
 
 const StatusIcon = ({ kind, size = 'medium', ...rest }: StatusIconProps) => {
-  let Component = Info;
+  let Component;
+  let ariaLabel;
 
   switch (kind) {
     case 'success':
       Component = CheckCircle;
+      ariaLabel = 'success';
```

Comment:
**suggestion:** Maybe `success icon`? Worried that the blind user won't know what `success` by itself means.